docs/_documentations/remotedeploy-combo.md

```diff
@@ -10,59 +10,64 @@ parent: installoncloud
 order: 2
 ---
 
-## Installing a Codewind remote deployment
+# Installing a Codewind remote deployment all-in-one
 
-Codewind includes a CLI to simplify the installation process. You will find `cwctl` in your HOME directory under the path ~/.codewind/{version}
+Codewind includes a CLI to simplify the installation process. You can find the `cwctl` CLI in your HOME directory under the path `~/.codewind/{version}`.
```

Comment:
- For conciseness, how about using the imperative, "Find the...", instead of "You can find the..."
- If/when we want to translate this file to another language, it's good to have the modifying words like "path" appear after the specifics like `~/.codewind/{version}`. So..."`~/.codewind/{version}` path."